lib/engine/game/g_1880/step/simple_draft.rb

```diff
@@ -39,10 +39,16 @@ def process_bid(action)
 
             @log << "#{player.name} chooses #{minor.full_name} (#{minor.name})"
 
+            assign_bcr_share_to_fi(action.entity, minor) if player.shares.find { |s| s.corporation.id == 'BCR' }
```

Comment:
Why do you need to do this? The shares ability should trigger float_corporation being called.

Reply:
float_corporation happens as soon as a player wins the private. At that point they do not own a FI so no share is sent to a FI. Once the player with the BCR picks a FI, then the share is assigned there. 
This code does this during the draft, as it should per the rules